packages/api-clients/open-api/bffApi.yml

```
@@ -16580,6 +16649,35 @@ components:
           format: uuid
       required:
         - id
+    DelegatedEservice:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          format: uuid
```

Since the model is called `DelegatedEservice` it's misleading that we have `id` and `eserviceId`: one would think that the `id` is the Eservice ID because this is the model for an Eservice - I would rename into `delegationId` or rename the model into CompactDelegationWithEservice or something similar

```suggestion
        delegationId:
          type: string
          format: uuid
```